conf/gulp.conf.js

```diff
@@ -50,7 +51,7 @@ for (const pathName in exports.paths) {
 exports.exitOnError = false;
 exports.errorHandler = function (title) {
   return function (err) {
-    gutil.log(gutil.colors.red(`[${title}]`), err.toString());
+    log(colors.red(`[${title}]`), err.toString());
```

Comment:
Just making sure that brackets have to be in a string here?

Reply:
the square brackets? or the back-ticks?

The back-ticks are for interpolation (see [template literals|https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals])
The square brackets were just here for still I think :)